Source/pages/knowMore.html

```diff
@@ -281,13 +295,196 @@ <h4 class="panel-title">
     </div>
 </div>
 
+<!-- tile Modal 4 -->
+<div class="tile-modal modal fade" id="tileModal4" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-content">
+        <div class="close-modal" data-dismiss="modal">
+            <div class="lr">
+                <div class="rl">
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2">
+                    <div class="modal-body">
+                        <!-- Project Details Go Here -->
+                        <h2>Benefits of Walking and Cycling</h2>          
+						<p>
+                            <div id="accordionCalories" role="tablist" aria-multiselectable="true">
+                                <div class="panel panel-default">
+                                    <div class="panel-heading" role="tab" id="headingOneCalories">
+                                        <h4 class="panel-title">
+                                            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOneCalories" aria-expanded="false" aria-controls="collapseOneCalories">
+                                                <i class="fa fa-plus accordian-icon"></i>
+                                                Health benefits
+                                            </a>
+                                        </h4>
+                                    </div>
+                                    <div id="collapseOneCalories" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOneCalories">
+		                                <ul>
+		                                    <li>Regular physical activity dramatically cuts the risk of developing heart disease, diabetes, obesity and some cancers. It also reduces cholesterol, lowers high blood pressure and is good for our sense of wellbeing.</li>
+											<li>it strengthens muscles, bones and joints.</li>
+											<li>it helps maintain flexibility and co-ordination as you get older.</li>
+		                                    <li>cycling raises your metabolic rate, which can help to keep weight off.</li>
+											<li>moderate pedal-pushing burns up to 500 calories per hour, which is more than walking or swimming.</li>
```

Comment:
Could you add a comment in the source code pointing to the source of these estimations?